src/Data/V/Linear.hs

```diff
@@ -36,17 +37,20 @@
 -- rather than linear types).
 module Data.V.Linear
   ( V,
+    next#,
+    next,
+    -- split,
+    Elim (..),
+    cons,
+    -- fromReplicator,
+    -- dupV,
```

Comment:
You should export these.

Reply:
As you found below, I can't export them because these functions do not typecheck yet (because of the `elim/make` mess).